infra/main.bicep

```diff
@@ -413,6 +416,7 @@ module web './app/web.bicep' = if (hostingModel == 'code') {
       AZURE_SEARCH_URL_COLUMN: azureSearchUrlColumn
       AZURE_SPEECH_SERVICE_NAME: speechServiceName
       AZURE_SPEECH_SERVICE_REGION: location
+      SPEECH_RECOGNIZER_LANGUAGES: recognizedLanguages
```

Comment:
We will need `SPEECH_RECOGNIZER_LANGUAGES: recognizedLanguages` in the container section too. Just an idea, but you could move the param and the app setting to this file /workspaces/chat-with-your-data-solution-accelerator/infra/app/web.bicep which will do it for code and container. I suppose the decision is, do you think many people will change this per deployment - again - I'm happy either way

Reply:
Thanks for pointing this out Ross, I've added this to container. 
I don't think language settings should be a part of the base settings, and rather should be per deployment. So since you said you're happy either way, I propose to keep the setting here rather than in web.bicep.